src/main/java/org/jabref/model/entry/FileFieldWriter.java

```diff
@@ -7,7 +7,7 @@
 
 public class FileFieldWriter {
 
-    private FileFieldWriter() {
+    FileFieldWriter() {
```

Comment:
Why is this constructor not private anymore?

Reply:
Was a relict from testing. Didn't see that it had static method on the first try